### DISCUSSION TITLE
[Feature Request] New Component Slider

### DISCUSSION THREAD

**Discussion by junevm** [heroui-inc/heroui#986] [Feature requests]
NextUI does not have a slider component.

**Comment by aztlan2k** [comment]
It does seem a bit odd that a slider component isn't available.  Most -- but not all -- other major component libraries provide a slider component.

**Reply by THE-SIMPLE-MARK** [reply]
I agree, I was trying to import it for a good minute or so

**Reply by cauta** [reply]
any sample for now?

**Comment by dougkeefe** [comment]
Just adding a +1 for slider.  

Are there any preferred UI solutions out there that this could model?

**Reply by dougkeefe** [reply]
https://github.com/nextui-org/nextui/pull/1686

**Comment by wingkwong** [comment]
https://www.heroui.com/docs/components/slider
